### ISSUE TITLE
Sync with Asp.Net

### ISSUE THREAD

**Issue by bhumika12123** [thekordy/ticketit#392]
Hello,

I have one asp.net application having couple of user(s) available. i want to sync this asp.net user's login with this (ticketit) application. When any user login to asp.net website, that is sync with ticketing system. Please help me out how to resolve this issue.

**Comment by balping** [comment]
Hi @bhumika12123 

This is completely a Laravel-related issue and has nothing to do with Ticketit specifically.

Start reading here: https://laravel.com/docs/5.5/authentication#authenticating-users

You should implement your own authentication logic, or you could use OAuth with your asp.net app as a server and using [Laravel Socialite](https://github.com/laravel/socialite) on the laravel side (with your own  provider).

A more proper solution would be to have a central LDAP server and use that both in laravel and asp.net. Here's a package: https://github.com/Adldap2/Adldap2-Laravel

So basically you have many choices. I hope I could provide some starting points, but I won't be able to provide further help
